azure/services/aso/aso.go

```diff
@@ -37,6 +37,9 @@ import (
 const (
 	// ReconcilePolicyAnnotation is the annotation key for ASO's reconcile policy mechanism.
 	ReconcilePolicyAnnotation = "serviceoperator.azure.com/reconcile-policy"
+	// PrePauseReconcilePolicyAnnotation is the annotation key for the value of
+	// ReconcilePolicyAnnotation that was set before pausing.
+	PrePauseReconcilePolicyAnnotation = "sigs.k8s.io/cluster-api-provider-azure-pre-pause-reconcile-policy"
```

Comment:
Why is the naming pattern of the annotation key different for `ReconcilePolicyAnnotation` and `PrePauseReconcilePolicyAnnotation` i.e `serviceoperator.azure.com` vs `sigs.k8s.io`?

Reply:
Makes sense. Thank you